crates/e2e/src/setup/colocation.rs

```diff
@@ -106,6 +101,8 @@ account = "{account}"
     let liquidity = liquidity.to_string(contracts);
     let config_file = config_tmp_file(format!(
         r#"
+encoding="domain"
```

Comment:
The logic to convert CoW AMM JIT orders is currently only available in the `domain` encoding implementation. Since we have been running this for a while I don't think we also need to implement it for the legacy logic.
AFAICS we can get rid of the legacy logic and make the new way the default.

Reply:
yes, sorry I have a PR to do that on my machine that I never put up for review. Will do that